src/managers/StageInstanceManager.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+const BaseManager = require('./BaseManager');
+const { TypeError, Error } = require('../errors');
+const StageInstance = require('../structures/StageInstance');
+const { PrivacyLevels } = require('../util/Constants');
+
+class StageInstanceManager extends BaseManager {
+  constructor(guild, iterable) {
+    super(guild.client, iterable, StageInstance);
+
+    /**
+     * The guild this manager belongs to
+     * @type {Guild}
+     */
+    this.guild = guild;
+  }
+
+  /**
+   * The cache of this Manager
+   * @type {Collection<Snowflake, StageInstance>}
+   * @name StageInstanceManager#cache
+   */
+
+  add(data, cache) {
+    return super.add(data, cache);
+  }
```

Comment:
Redundant

Reply:
yup, that's a leftover. There was more there but it got removed later. Fixed in the latest commit 👍